### ISSUE TITLE
Question about stream analysis

### ISSUE THREAD

**Issue by KingTuring** [fraunhoferhhi/vvenc#174]
Hi!
I want to check the mode information of stream encoded by vvenc, but when I check it with software called streameye, the code stream cannot be opened. Stream can normally analyze the code stream of vtm12.0. So I have the following two questions:
1. Which version of VTM corresponds to vvenc 1.4, that is, which version of VTM's stream is the same with vvenc1.4?
2. Is there any software like streameye that can analyze the stream generated by vvenc？

Thanks in advance,
Jin.

**Comment by adamjw24** [comment]
Hi,

its interesting, because this year at NAB I was at the Elecard stand, and we had a look at our vvenc-1.4 bitstreams in their demo software. It worked alright.

Your question misses the point, that both software are compliant VVC encoders. VTM since version 11.0 (10.0 was supposed to be the final version, but had a major bug). VVenC since version 0.1 is supposed to be VVC compliant. There is still the potential that VVenC has a bug. So if the bitstream cannot be opened, you could try decoding it with VTM decoder (or VVdeC, but if it doesnt decode with VVdeC the next step would be to try VTM either way). In this case please file a bug report, specifying the exact used encoder version, parameters, some info about the input sequence and possibly the bitstream.

So in simple terms, any vvenc version is just as VVC compliant as any VTM version including and after 11.0.

Hope this helps, Adam

**Comment by KingTuring** [comment]
Hi,
I have confirmed that both vvdec and VTM can normally decode the stream generated by vvenc, so the problem should come from the streameye software itself. 
Thank you for your patience.
Jin.
